### PR TITLE
Update build deps

### PR DESCRIPTION
**Pull request by evertiro**
libsass version should now be libsass1, and theme names have been changed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,9 +68,9 @@ r
  ```
  sudo apt remove system76-pop-gtk-theme
  sudo make uninstall
- sudo rm -rf /usr/share/themes/{Pop,Pop-Eta,Pop-Nokto,Pop-Nokto-Eta}
- rm -rf ~/.local/share/themes/{Pop,Pop-Eta,Pop-Nokto,Pop-Nokto-Eta}
- rm -rf ~/.themes/{Pop,Pop-Eta,Pop-Nokto,Pop-Nokto-Eta}
+ sudo rm -rf /usr/share/themes/Pop{,-dark{,-slim},-light{,-slim},-slim{,-dark,-light}}
```
Comment (isantop) [inline README.md]: These are actually leftover from when the theme was still based on Adapta. It covers the case that someone is using a very old (Version 1) copy of the theme. `sudo make uninstall` will remove the current version.

If we want to change this around, it should probably just be removed instead.

**Comment by isantop** [comment]
@evertiro Thankd for taking that on! Just a couple of small notes.

File: README.md (hunk 1 above)
Comment (evertiro) [reply README.md]: Is it possible that someone could still be using Version 1? If so, maybe we should rework the readme instructions and leave the `rm` as it was.

Comment (evertiro) [reply README.md]: Removed the redundant dependency though.

Comment (isantop) [reply README.md]: It'll always be possible that someone is still using V1, although it is unlikely.

Comment (jackpot51) [reply README.md]: This change still needs to be reverted

**Pull request closed without merging**